### PR TITLE
Web frontend navigation

### PR DESCRIPTION
**Pull request by t-animal**

### CODE REVIEW

File: frontend-web/src/scaffolding/header-and-logo.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Component, ParentComponent } from "solid-js";
+
+
```
Comment (mxwnk) [inline frontend-web/src/scaffolding/header-and-logo.tsx:3]: Minor: format

File: frontend-web/src/coffee-call/coffee-call.module.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.buttonRow>*:not(:first-child) {
```
Comment (mxwnk) [inline frontend-web/src/coffee-call/coffee-call.module.css]: I was a little bit confused by the word `module`. Why do we use it in the css file names?

**Review by mxwnk — APPROVED** [review]

File: frontend-web/src/scaffolding/header-and-logo.tsx (hunk 1 above)
Comment (t-animal) [reply frontend-web/src/scaffolding/header-and-logo.tsx:3]: Yes, the whole project still is a mess when it comes to formatting. I'd rather format everything in one go at some point, though

File: frontend-web/src/coffee-call/coffee-call.module.css (hunk 2 above)
Comment (yogan) [reply frontend-web/src/coffee-call/coffee-call.module.css]: CSS modules are actually a thing™ and different from "regular" CSS files:
https://github.com/css-modules/css-modules

I'm not an expert myself, but the main takeaway is: a CSS module (marked by the name suffix)
is scoped to the component that imports it. E.g. when you style `h1` as `red` in some module,
it does not affect all `h1`s on the page, but only those in the component.

**Comment by yogan** [comment]
I found a small layout bug, the buttons are jumping around as soon as messages appear, which is due to the `main` growing:

jumpy-buttons

I'll push a small fix for that soon.

**Review by yogan — APPROVED** [review]
LGTM. Just give me a few minutes to fix the positioning of the buttons.

File: frontend-web/src/coffee-call/coffee-call.module.css (hunk 2 above)
Comment (t-animal) [reply frontend-web/src/coffee-call/coffee-call.module.css]: https://glenmaddern.com/articles/css-modules We're not using it entirely correctly at this point, though, imho. I personally find it a wee bit annoying for such a small project, but it's used by default in solid.

**Pull request merged**